modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSView.tsx

```diff
@@ -140,6 +141,11 @@ const FDSView = ({
 						fdsView={fdsView}
 						fdsViewsURL={fdsViewsURL}
 						namespace={namespace}
+						onFDSViewUpdate={(
+							updatedViewData: FDSViewType
+						): void => {
+							setFDSView({...fdsView, ...updatedViewData});
+						}}
```

Comment:
Do we need to specify the type here? I assume that specifying it above on line 60 makes it unnecessary to specify on line 145-146

Reply:
Probably not. Just inlined the function from an external declaration and forgot to remove.